gcloud/gcloud_kms_integration_test.go

```diff
@@ -80,6 +83,13 @@ func (i *gcloudSecretTest) TestPutSecret(t *testing.T) error {
 	assert.NoError(t, err, "Unexpected error on PutSecret")
 	i.totalPuts++
 
+	// Put Secret with large secret data text
+	i.largePassphrase = randSeq(2000)
+	secretData[testSecretIdWithLargePassphrase] = i.largePassphrase
+	err = i.s.PutSecret(testSecretIdWithPassphrase, secretData, keyContext)
```

Comment:
This Put needs to be with the testSecretIdWithLargePassphrase key

Reply:
good catch. updated.